doc/BUILDSERVER.md

```diff
@@ -0,0 +1,98 @@
+# Buildserver
+
+## Introduction
+Most of the build jobs on the jenkins server that handles automated testing of
+contributions have been replaced by a single
+[job](https://github.com/ElektraInitiative/libelektra/).
+
+
+## Jenkinsfile
+Jenkinsfiles describe what actions the build system should execute on what
+build slave.
+Currently Elektra uses two different files.
+
+One is for daily maintanence tasks, like cleaning up build servers
+([Jenkinsfile.daily](https://github.com/ElektraInitiative/libelektra/blob/master/scripts/jenkins/Jenkinsfile.daily))
+.
+The other one is triggered on code changes and is for testing changes to the
+codebase
+([Jenkinsfile](https://github.com/ElektraInitiative/libelektra/blob/master/scripts/jenkins/Jenkinsfile))
+.
+
+The language used is a groovy based DSL described in the
+[Jenkinsfile book](https://jenkins.io/doc/book/pipeline/jenkinsfile/).
+Most groovy syntax is allowed to describe pipelines in a Jenkinsfile, but it
+is executed in a sandbox which might block certain calls.
+
+Since plugins might extend the pool of available commands or variables a full
+list of currently available syntax can be seen in
+[pipeline
+syntax](https://build.libelektra.org/jenkins/job/libelektra/pipeline-syntax/)
+after a login to the build server.
+
+## Security
+Since a malicious PR could easily destroy the build server and slaves or expose
```

Comment:
Merge this with content from doc/GIT.md?

From my point of view we can stop having content about the buildserver in doc/GIT.md

Reply:
done in 90a7bb4